database/redis/redis.def

```diff
@@ -0,0 +1,30 @@
+Bootstrap:library
+From: debian:9
+
+%help
+
+	Singularity container running Redis 5.0.0 for Debian 9 (Stretch)
+
+%post
+
+	#Update and install dependencies
+	apt-get update
+	apt-get install -y wget build-essential
+
+	#Get redis and install it
+	wget http://download.redis.io/releases/redis-5.0.0.tar.gz
+	tar xzf redis-5.0.0.tar.gz
+	cd redis-5.0.0
+	make
+
+%startscript
+
+	#Start the server daemon with this command
+	cd /redis-5.0.0/src/
+	./redis-server
```

Comment:
Instead of the `cd` command,
can you replace it will `/redis-5.0.0/src/redis-server` or `/redis-5.0.0/src/./redis-server`?